paper-drawer-panel.css

```diff
@@ -107,10 +108,17 @@ iron-selector > #main {
 
 .right-drawer.narrow-layout > #drawer:not(.iron-selected) {
   left: auto;
+  /* paper-drawer-panel/issues/59 */
+  visibility: hidden;
+
   -webkit-transform: translateX(100%);
   transform: translateX(100%);
 }
 
+.right-drawer.narrow-layout.dragging > #drawer:not(.iron-selected)  {
+  visibility: visible;
+}
+
```

Comment:
We need to make the drawer visible when you tap on the edge and the drawer is supposed to slide out a bit.